week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -18,23 +18,44 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
+// calling this function should pour your bottle full (volume = 100);
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp: function () {
+    this.volume += 100;
   },

```

Comment:
At this part of the function, you have been asked to set the volume of the bottle to 100 not increment by 100. It needs to be fixed.

Reply:
done!